proposals/0316-Clarification-of-transport-switching-rules-and-improvement-of-UX.md

```diff
@@ -6,60 +6,72 @@
+OEMs can implement processes related to primary transport switching, such as message display and screen transition, according to the HMI specifications. They can also eliminate the inconvenience for the users when using the primary transport switch.
 
 New RPC: `OnDeviceSwitchStatus`(HMI_API.xml)
 
 ```xml
 <function name="OnDeviceSwitchStatus" messagetype="notification">
-    <description> Notify HMI of the current state of the transport switch </description>
+    <description> Notify HMI of the current state of the primary transport switch </description>
     <param name="status" type="Common.SwitchStatus" mandatory="true"/>
+    <param name="switchingTransportResult" type="Common.SwitchingTransportResult" mandatory="false"/>
 </function>
 
+<struct name="SwitchingTransportResult" mandatory="false"/>
+   <param name="switchingResult" type="SwitchResult" mandatory="true"/>
+   <param name="failReason" type="Strings" mandatory="false"/>
+</struct>
+
 <enum name="SwitchStatus">
-   <element name="Started"/>
-   <element name="Finished"/>
+   <element name="STARTED"/>
+   <element name="FINISHED"/>
+</enum>
+
+<enum name="SwitchResult">
+   <element name="PASSED"/>
+   <element name="FAILED"/>
 </enum>
 ```
 
 The below shows the new RPC (in red text) added to the existing logic. It is necessary to add the related information regarding the new RPC, including this sequence diagram, to the "SDL Core Guides" and "HMI Documentation" on SDL Developer Portal (smartdevicelink.com).
 
-*1 The switching from BT to USB logic in the same device transport of iOS is already approved and implemented by the SDLC in SDL 0053 - Connectivity via iAP-BT and Transport Switch. See [PR](https://github.com/smartdevicelink/sdl_evolution/issues/158) and [sequence](https://cloud.githubusercontent.com/assets/11158516/26067529/3f428434-39a3-11e7-910b-6cc4069e0be3.png).<br>
+*1 The switching from BT to USB logic in the same device primary transport of iOS is already approved and implemented by the SDLC in SDL 0053 - Connectivity via iAP-BT and Transport Switch. See [PR](https://github.com/smartdevicelink/sdl_evolution/issues/158) and [sequence](https://cloud.githubusercontent.com/assets/11158516/26067529/3f428434-39a3-11e7-910b-6cc4069e0be3.png).<br>
 *2 With the current SDL standard specification, the device cannot identify whether it is the same device or not. Therefore, device identification by the HU must be a prerequisite.
 
 <b>Figure 1.</b> Sequence with new RPC added to existing logic
 
-![Figure_1_Sequence_with_new_RPC_added_to_existing_logic.png](../assets/proposals/0316-Clarification-of-transport-switching-rules-and-improvement-of-UX/Figure_1_Sequence_with_new_RPC_added_to_existing_logic.png)
+![Figure_1_Sequence_with_new_RPC_added_to_existing_logic.png](../assets/proposals/0316-Clarification-of-transport-switching-rules-and-improvement-of-UX/Figure_1_Sequence_with_new_RPC_added_to_existing_logic1.png)
```

Comment:
I've found a few required changes to this image:
- "unregisteration" should be "unregistration" in bottom right corner.
- "Guard" in bottom left corner should be removed.
- "Reconnection timer IS expired" condition should be added to bottom left corner (see: https://cloud.githubusercontent.com/assets/11158516/26067529/3f428434-39a3-11e7-910b-6cc4069e0be3.png).

Reply:
@theresalech -san,
Thank you for your advise.
I updated the image file.
Could you please confirm it.
Thank you.